server/autotest_server/testers/r/r_tester.py

```diff
@@ -0,0 +1,102 @@
+import subprocess
+import os
+import json
+from typing import Dict, Optional, IO, Type, List, Union
+
+from ..tester import Tester, Test, TestError
+from ..specs import TestSpecs
+
+
+class RTest(Test):
+    def __init__(
+        self,
+        tester: "RTester",
+        test_file: str,
+        result: Dict,
+        feedback_open: Optional[IO] = None,
+    ) -> None:
+        """
+        Initialize a R test created by tester.
+
+        The result was created after running the tests in test_file and test feedback
+        will be written to feedback_open.
+        """
+        self._test_name = f"{test_file}:{result.get('context', '')}:{result['test']}"
```

Comment:
After running some sample tests (from https://github.com/MarkUsProject/Markus/pull/5551/files), it seems `result.get('context', '')` returns `None` (?), leading to test names like `test.R:None:negative integers can be added`. I'd rather not have the "None" appear, since I'm worried this will be confusing for students and instructors. For R testing, top-level tests will be the norm.

Can we change this so that if this field is None or blank, we report the test name as `f"{test_file}:{result['test']}"`?